app/code/Magento/Ui/view/base/web/js/modal/prompt.js

```diff
@@ -154,8 +154,7 @@ define([
          */
         closeModal: function (result) {
```

Comment:
Modify [these](https://github.com/magento/magento2/pull/25349/files#diff-9458ab0c3f9705f0f9bdb6fc1b15907dR47-R68) functions  to accept the event as an argument and use it in closeModal function call as a first argument(same as in [confirm.js](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Ui/view/base/web/js/modal/confirm.js#L40-L60)). After that, start accepting in closeModal event as first argument and result as second([example](https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Ui/view/base/web/js/modal/confirm.js#L89)).